### ISSUE TITLE
JSDoc3 `~` namepaths are not supported

### ISSUE THREAD

**Issue by SevInf** [Kuniwak/jsdoctypeparser#13]
JSDoc3 allows to use `~` character to refer to the inner member of the class/namespace ([docs](http://usejsdoc.org/about-namepaths.html)), but `jsdoctypeparser` does not recognize such declarations as a valid types:

```
>parser.parse('Name~property')
Unexpected token: "":
Name~property
   ^
```

**Comment by Kuniwak** [comment]
Thanks for your reporting, @SevInf .

I'm gonna enable it on v2.0.0 (related 9e60e6d).
And after v2.0.0, we can get new AST that is not comatible to v1.x.

**Comment by Kuniwak** [comment]
It will be released with v2.0.0
